Amazon.IonObjectMapper/Amazon.IonObjectMapper.csproj

```diff
@@ -13,6 +13,8 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>amazon api aws deserialize ion json serialize</PackageTags>
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageIcon>amazon.png</PackageIcon>
+    <PackageIconUrl />

```

Comment:
Do we need this unspecified PackageIconUrl?

Reply:
Its generated by default so better safe than sorry